src/components/generateKeys/GenerateKeypair.tsx

```diff
@@ -0,0 +1,182 @@
+import { useState } from 'react';
+
+interface initialErrorState {
+  status: boolean;
+  message: string;
+}
+
+function GenerateKeypair() {
+  const [storedKeypair, setStoredKeypair] = useState<boolean>(false);
+  const [isCopied, setIsCopied] = useState<boolean>(false);
+  const [error, setError] = useState<initialErrorState>({ status: false, message: '' });
+
+  const textToCopy = `Public key:
+  GDRU64ATGPDM2EVNOIVVM3P3X55LT5FLUA6XMD55IYACPDDVUWZ5BQTW
+  Secret key:
+  SBZK3IZGHIM3GP2I4K32QVSWBNVUAD2KL6VMUQOQEMXAIWNXDU7KRTAD
+  `;
+
+  const toggleStoredKeypair = () => {
+    if (!storedKeypair) {
+      setError({ status: false, message: '' });
+    }
+
+    setStoredKeypair(!storedKeypair);
+  };
+
+  const copyToClipboard = (textToCopy: string) => {
+    navigator.clipboard.writeText(textToCopy).then(
+      () => {
+        setIsCopied(true);
+        setTimeout(() => {
+          setIsCopied(false);
+        }, 2000);
+      },
+      (err) => {
+        console.log('failed to copy', err.mesage);
```

Comment:
Not everyone knows about the DevTools console. We should show the user this message.